icons/eventbrite.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Eventbrite icon</title><path d="M10.5 5.8c2.7-.6 5.3.5 6.8 2.5L5.6 11c.4-2.5 2.3-4.6 4.9-5.2zm6.9 9.8c-.9 1.3-2.3 2.3-3.9 2.6-2.7.6-5.3-.5-6.8-2.6L18.4 13l1.9-.4 3.7-.8c0-.8-.1-1.6-.3-2.3C22.2 3 15.8-.9 9.3.6S-1.2 8.4.3 14.7 8.2 25 14.7 23.5c3.8-.9 6.8-3.4 8.2-6.6.1-.1-5.5-1.3-5.5-1.3z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Eventbrite icon</title><path d="M10.5084 5.7562c2.699-.6 5.2968.5 6.7958 2.4989l-11.6927 2.698c.4-2.498 2.299-4.597 4.8969-5.197zm6.8958 9.7937c-.9 1.3-2.298 2.299-3.8969 2.599-2.699.5999-5.2968-.5-6.7958-2.599l11.6917-2.598 1.9-.4L24 11.752c0-.799-.1-1.598-.3-2.298C22.201 2.9584 15.8052-.9385 9.3104.5604 2.8126 2.0593-1.1843 8.3551.3147 14.65c1.499 6.2968 7.8948 10.2937 14.3896 8.7948 3.7979-.9 6.7968-3.398 8.1957-6.5958.1-.1-5.4968-1.299-5.4968-1.299z"/></svg>
```

Comment:
Seems to be a slight difference between PR (red) and source (black).
Path direct from source is below.
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 98"><path d="M44.053 23.401c10.985-2.481 21.946 2.016 28.049 10.52L23.556 44.89C25.23 34.627 33.068 25.882 44.053 23.4zm28.32 40.35c-3.67 5.322-9.329 9.35-16.227 10.908-11.057 2.498-22.09-2.076-28.169-10.69l48.698-11.002 7.931-1.792 15.165-3.426a47.9 47.9 0 00-1.263-9.656C92.336 11.931 65.658-4.382 38.923 1.658 12.188 7.698-4.482 33.804 1.691 59.967c6.172 26.162 32.85 42.475 59.585 36.435 15.753-3.559 28.007-14.085 34.156-27.484l-23.059-5.167z" fill-rule="evenodd"></path></svg>
```

![image](https://user-images.githubusercontent.com/1830867/102893313-4bd59a00-4459-11eb-875d-e6375b480470.png)

Reply:
@adamrusted Fixed. Thanks for your review 🌟